package.json

```diff
@@ -50,7 +50,7 @@
 		"@graphql-tools/url-loader": "^7.12.1",
 		"@kiva/kv-components": "^3.7.1",
 		"@kiva/kv-tokens": "^2.4.0",
-		"@mdi/js": "^5.9.55",
+		"@mdi/js": "^7",
```

Comment:
What did this add? We'll need to update in cms-page-server as well...

Reply:
new icons!

Reply:
:)